Source/FieldSolver/WarpXPushFieldsEM.cpp

```diff
@@ -418,6 +418,8 @@ WarpX::PushPSATD ()
     if (pml_rz[0]) pml_rz[0]->PushPSATD(0);
 #endif
 
+    if (WarpX::do_dive_cleaning) PSATDForwardTransformF();
+    if (WarpX::do_divb_cleaning) PSATDForwardTransformG();
```

Comment:
@dpgrote @RemiLehe I think these two lines might have been added by mistake when merging `development` into this branch. Do you agree?

Reply:
Never mind, this was showing up as a change/commit "since my last review", but it does not show up in the final file changes if I look at those separately. I guess everything is good.